.github/workflows/main.yml

```
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '*'
+  pull_request:
+    branches:
+      - master
+      - '*'
```

I think you can just replace all this with?

```suggestion
on: push
```

---

it seems to work, but the test results aren't conveniently shown in the PR summary for review and validation